src/main/resources/application.yaml

```diff
@@ -54,7 +54,7 @@ idam:
     secret: ${IDAM_OAUTH2_CLIENT_SECRET:QM5RQQ53LZFOSIXJ}
     redirect_uri: ${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authenticated}
 
-allowed-services-for-callback : ccd_data,sscs_bulkscan,bulk_scan_processor,bulk_scan_orchestrator  # comma separated list of service names
+allowed-services-for-callback : ccd_data,sscs_bulkscan,bulk_scan_processor,bulk_scan_orchestrator,sscs  # comma separated list of service names
```

Comment:
question: does `sscs` relate to an actual service here? What's the context

Reply:
It is. It will allow tribunals-api to call bulk-scan authenticating as the sscs service